Group 13 Final Project

SukiyAkiOhno opened the pull request.

SukiyAkiOhno committed:
Create proposal.md

charlieroberts commented:
This sounds great! I saw you were thinking about using Trace.moe... does this mean users will also upload screenshots to your database? This wasn't mentioned in your proposal. Maybe you could use the Kitsu.io api to grab images of movie posters associated with each movie? Or maybe there's another API out there to use for this. 

I think this is very doable, but I would like to see images for the movies in the app (if you weren't already planning on doing this). Good luck!

YangLyuJimmy committed:
Add files via upload

YangLyuJimmy committed:
Merge pull request #1 from SukiyAkiOhno/vue-starter-code

Add files via upload

YangLyuJimmy committed:
add node-module

YangLyuJimmy committed:
Create .gitignore

YangLyuJimmy committed:
new one

YonghuaWang706 committed:
Merge pull request #2 from SukiyAkiOhno/jimmy

good jb

aamaxaa committed:
nuxt

aamaxaa committed:
Merge branch 'maxine'

aamaxaa committed:
Delete node_modules directory

aamaxaa committed:
Delete .nuxt directory

aamaxaa committed:
Delete assets directory

aamaxaa committed:
Delete components directory

aamaxaa committed:
Delete layouts directory

aamaxaa committed:
Delete pages directory

aamaxaa committed:
Cleared

aamaxaa committed:
setup

YangLyuJimmy committed:
add about us page

YangLyuJimmy committed:
Create 网址参考.txt

YangLyuJimmy committed:
Merge pull request #3 from SukiyAkiOhno/jimmy

add about us page, and 网址参考txt

YonghuaWang706 committed:
delete_all

YonghuaWang706 committed:
new repo

YonghuaWang706 committed:
simple_server_with_vue_demo

SukiyAkiOhno committed:
Merge branch 'yonghua'

YangLyuJimmy committed:
do some little test to check my understanding

YangLyuJimmy committed:
Merge pull request #4 from SukiyAkiOhno/jimmy

do some little test to check my understanding

YangLyuJimmy committed:
create simple nav bar

YangLyuJimmy committed:
Merge pull request #5 from SukiyAkiOhno/jimmy

create simple nav bar

YonghuaWang706 committed:
with_bottstrap

YonghuaWang706 committed:
Revert "with_bottstrap"

This reverts commit 6a497bdf89e8ef8c6cdc49aba589b55633307dee.

YonghuaWang706 committed:
Revert "Revert "with_bottstrap""

This reverts commit b363f1e7536e9a2a9f71ef9c33a5b5d3d1db5ded.

YonghuaWang706 committed:
login_ready

YonghuaWang706 committed:
Merge pull request #6 from SukiyAkiOhno/yonghua

普通登录功能ready

YangLyuJimmy committed:
final starter code

SukiyAkiOhno committed:
Update server.js

SukiyAkiOhno committed:
Update watchlist needed stuff

YangLyuJimmy committed:
Merge pull request #8 from SukiyAkiOhno/Suki

Suki

YonghuaWang706 committed:
github_wowrking

YangLyuJimmy committed:
navbar and clear main.js

1. deleted main.js which contains unnecessary vue
2. add nav.html, in each page, just add import jquery and add a div and import function

jingyu-xie committed:
add profile page (incompleted)

jingyu-xie committed:
Merge pull request #9 from SukiyAkiOhno/profile

add profile page (incompleted)

YonghuaWang706 committed:
search_engine

YonghuaWang706 committed:
resolve_conflicts_server

YonghuaWang706 committed:
picture_in_search

YangLyuJimmy committed:
add about us page

available to send email for feedback

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

SukiyAkiOhno committed:
Using light mode only

YangLyuJimmy committed:
add some instructions and dog icon

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

YangLyuJimmy committed:
fix some small bug with v-text-field

YangLyuJimmy committed:
add greetings sentence at index.html

jingyu-xie committed:
move profile to index

jingyu-xie committed:
Merge pull request #10 from SukiyAkiOhno/profile

move profile to index, and add show username at the index page

jingyu-xie committed:
some debug

jingyu-xie committed:
Merge pull request #11 from SukiyAkiOhno/profile

some debug

SukiyAkiOhno committed:
Update watchlist.html

new table sort view

SukiyAkiOhno committed:
add js for watchlist

SukiyAkiOhno committed:
Update server.js

Some unsuccessful sort into a different table(

YangLyuJimmy committed:
fix email sender and cc order

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

SukiyAkiOhno committed:
Update server.js

jingyu-xie committed:
add change password function

If you are using github, there will be a reminder that "you cannot change password because you login with GitHub"

jingyu-xie committed:
Merge pull request #12 from SukiyAkiOhno/profile

add change password function

SukiyAkiOhno committed:
Update server.js and watchlist.js

SukiyAkiOhno committed:
Add & Remove function for Watchlist

aamaxaa committed:
KannaDrink

SukiyAkiOhno committed:
Still need to fix

SukiyAkiOhno committed:
1

SukiyAkiOhno committed:
UPDATE EDIT FUNCTION

SukiyAkiOhno committed:
Merge pull request #13 from SukiyAkiOhno/Suki-with-edit-function

Suki with edit function

jingyu-xie committed:
attribute almost done, 另外jimmy你图崩了

jingyu-xie committed:
Merge pull request #14 from SukiyAkiOhno/profile

attribute almost done, 另外jimmy你图崩了

jingyu-xie committed:
push again，不知道上次出什么问题了

jingyu-xie committed:
Merge pull request #15 from SukiyAkiOhno/profile

push again，不知道上次出什么问题了

YangLyuJimmy committed:
fix the about us image problem

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

jingyu-xie committed:
attribute generator done !

jingyu-xie committed:
Merge pull request #16 from SukiyAkiOhno/profile

attribute generator done !

jingyu-xie committed:
update watchlist

- 保证watchlist的名字和评分不是空的
- add new watchlist和edit 打开的是弹窗

jingyu-xie committed:
Merge pull request #17 from SukiyAkiOhno/profile

update watchlist

jingyu-xie committed:
。。。

jingyu-xie committed:
Merge pull request #18 from SukiyAkiOhno/profile

。。。

aamaxaa committed:
Update kannadrink.png

jingyu-xie committed:
美化

jingyu-xie committed:
Merge pull request #19 from SukiyAkiOhno/profile

美化

jingyu-xie committed:
华哥update

jingyu-xie committed:
加了一个pie chart

jingyu-xie committed:
Merge pull request #20 from SukiyAkiOhno/profile

加了一个pie chart

YangLyuJimmy committed:
update port to 3000 and delete unnecessary files

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

jingyu-xie committed:
update score

jingyu-xie committed:
Merge pull request #21 from SukiyAkiOhno/profile

update score

jingyu-xie committed:
watchlist 小bug

jingyu-xie committed:
Merge pull request #22 from SukiyAkiOhno/profile

watchlist 小bug

SukiyAkiOhno committed:
Update each page's name

SukiyAkiOhno committed:
Few details

aamaxaa committed:
Embellishment Kanna

jingyu-xie committed:
update watchlist.js

jingyu-xie committed:
Merge pull request #23 from SukiyAkiOhno/profile

update watchlist.js

YangLyuJimmy committed:
add set email type

SukiyAkiOhno committed:
Update README.md

jingyu-xie committed:
Update index.html

jingyu-xie committed:
Merge pull request #24 from SukiyAkiOhno/profile

Update index.html

YangLyuJimmy committed:
123

YangLyuJimmy committed:
Merge branch 'main' of https://github.com/SukiyAkiOhno/final_project

Commit:
Updated with Glitch

YangLyuJimmy committed:
Merge pull request #25 from SukiyAkiOhno/glitch

Updated with Glitch

SukiyAkiOhno commented:
Group 13 finished!
